Classes/Notifications/NotificationsViewController.swift

```diff
@@ -47,53 +64,76 @@ FlatCacheListener {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        resetLeftBarItem()
-        if !showRead {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(
-                title: NSLocalizedString("Archives", comment: ""),
+        resetRightBarItem()
```

Comment:
Whyyyyy are you swapping the buttons?

Reply:
Yup. In the repo notifications view, the left button will be "back". So I'm moving the "mark all" to the right so that it's consistent between views.